facepy/signed_request.py

```diff
@@ -72,15 +79,14 @@ def __init__(self, signed_request=None, application_secret_key=None, application
         )
 
     def fetch_user_data_and_token(self):
-        from urlparse import parse_qs
         from . import GraphAPI, get_application_access_token
 
         app_token = get_application_access_token(self.application_id, self.application_secret_key, api_version=self.api_version)
         graph = GraphAPI(app_token, version=self.api_version)
 
         qs = graph.get('oauth/access_token', code=self.raw['code'], redirect_uri='', client_id=self.application_id, client_secret=self.application_secret_key)
-        self.raw['oauth_token'] = parse_qs(qs)['access_token'][0]
-        self.raw['expires'] = time.time() + int(parse_qs(qs)['expires'][0])
+        self.raw['oauth_token'] = qs['access_token']
```

Comment:
I think we're good! It's been a long time since Facebook did this, so I'm pretty sure that version is deprecated or even removed now.